scripts/pause.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# This script will pause all instances that have data = 1.
+# NOTE: User must have MAINNET_RPC_URL, PAUSER_ADDRESS, and PAUSER_KEY set in .env file.
+#
+# Example usage: bash scripts/pause.sh
+
+source .env
+
+# Set the network
+RPC_URL=$MAINNET_RPC_URL
+
+# Hyperdrive Registry contract address
+HYPERDRIVE_REGISTRY_ADDRESS="0x03f6554299acf544ac646305800f57db544b837a"
```

Comment:
this probably shouldn't be hardcoded, would leave this as an input environment variable